.devcontainer/image/Dockerfile

```
@@ -81,6 +81,9 @@ RUN apt-get update && \
     python3-pygments \
     # Required to embed git metadata into PDF from within Docker container:
     git
```

```suggestion
    git \
```

Needs a line continuation.

I already did this locally and pushed the image, hence `alexpovel/latex` is now online with `pdfunite` available.